src/parser/shadowlands/modules/covenants/kyrian/EffusiveAnimaAccelerator.ts

```diff
@@ -11,17 +11,17 @@ interface SpellInfo {
   name: string;
 }
 
-const SPELLS_BY_CLASS: { [key: number]: SpellInfo } = {
+const SPELLS_BY_CLASS: { [key: number]: SpellInfo | undefined } = {
   [CLASSES.DEATH_KNIGHT]: SPELLS.SHACKLE_THE_UNWORTHY,
   [CLASSES.DEMON_HUNTER]: SPELLS.ELYSIAN_DECREE,
   [CLASSES.DRUID]: SPELLS.KINDRED_SPIRITS,
   [CLASSES.HUNTER]: SPELLS.RESONATING_ARROW,
   [CLASSES.MAGE]: SPELLS.RADIANT_SPARK,
-  [CLASSES.MONK]: SPELLS.WEAPONS_OF_ORDER,
+  [CLASSES.MONK]: SPELLS.WEAPONS_OF_ORDER_CAST,
   [CLASSES.PALADIN]: SPELLS.DIVINE_TOLL,
   [CLASSES.PRIEST]: SPELLS.BOON_OF_THE_ASCENDED,
   [CLASSES.ROGUE]: SPELLS.ECHOING_REPRIMAND,
-  [CLASSES.SHAMAN]: SPELLS.VESPER_TOTEM,
+  [CLASSES.SHAMAN]: undefined,
```

Comment:
If you want to add the spell it should be 324386

Reply:
its 1am and i wasn't planning to be on rn so i'm just YOLOing that through once i confirm tests pass. can get that fixed once someone wants to do rsham things

Reply:
actually, fuck it, i'll do it right

Reply:
Yeah. I was just putting it out there if you wanted to do it haha
Its late just go to sleep haha

Reply:
![image](https://user-images.githubusercontent.com/25177817/163701064-729acaa6-7df4-409f-b59a-d04a98fc6ea1.png)
it appears i've been sniped